renderers/team_renderer.py

```diff
@@ -61,7 +61,8 @@ def render_team_details(cls, handler, team, year, is_canonical):
                 qual_avg = None
                 elim_avg = None
                 wlt = EventHelper.calculateTeamWLTFromMatches(team.key_name, event_matches)
-                year_wlt_list.append(wlt)
+                if event.official:
```

Comment:
This doesn't work. Certain offseason events that use the FRC API count as "official" too. Kind of awkward but unfortunately that's how it is. You should create a new set in https://github.com/the-blue-alliance/the-blue-alliance/blob/master/consts/event_type.py called something like `OFFICIAL_SEASON_EVENT_TYPES` and check that the event type is in that set.